docs/quick-release-notes/main.go

```diff
@@ -0,0 +1,314 @@
+				releaseVersionHasTag = true
+				tagVersionForMap = tagVersion.Name
+			}
+		}
+		if releaseVersionHasTag == false {
+			fmt.Println(redColor+"WARNING: release", releaseVersion, "missing tagInfo"+resetColor)
+		}
+		if releaseVersionHasTag {
+			releaseVersionAndTag[releaseVersion] = tagVersionForMap
+			releaseVersionAndNote[releaseVersion] = validationData.releaseNotes[i]
+		}
+	}
+	tagAndReleaseVersion := make(map[string]string)
+	for _, tagVersion := range validationData.tags[:] {
+		releaseVersionForMap := ""
+		tagHasVersionNumber := false
+		for _, releaseVersion := range validationData.changeLogInfo.versionNumbers[:] {
+			if tagVersion.Name == releaseVersion {
+				tagHasVersionNumber = true
+				releaseVersionForMap = releaseVersion
+			}
+		}
+		if tagHasVersionNumber == false {
+			fmt.Println(redColor+"WARNING: tag", tagVersion.Name, "missing changelog entry"+resetColor)
+		}
+		if tagHasVersionNumber {
+			tagAndReleaseVersion[tagVersion.Name] = releaseVersionForMap
+		}
+	}
+	return releaseData{releaseVersionAndNote: releaseVersionAndNote, releaseVersionAndTag: releaseVersionAndTag, tagAndReleaseVersion: tagAndReleaseVersion}
+}
+
+func splitAndEncodeURLPath(urlPath string) (string, string) {
+	urlPathSplit := strings.Split(urlPath, "/")
+	owner, name := url.QueryEscape(urlPathSplit[0]), url.QueryEscape(urlPathSplit[1])
+	return owner, name
+}
+
+func getReleases(authToken string, repoPath string) []listOfReleasesForUpdate {
+	repoOwner, repoName := splitAndEncodeURLPath(repoPath)
+	releasePath := fmt.Sprintf("https://api.github.com/repos/%v/%v/releases", repoOwner, repoName)
+	req, err := http.NewRequest("GET", releasePath, nil)
+	req.Header.Set("Accept", "application/vnd.github.v3+json")
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Authorization", "token "+authToken)
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		log.Fatal(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var eachRelease []listOfReleasesForUpdate
+	err = json.Unmarshal(body, &eachRelease)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return eachRelease
+}
+
+func getTagsFromAPI(tagsRepoPath string) []tagInfo {
+	// https://docs.github.com/en/rest/reference/repos#list-repository-tags
+	tagsRepoOwner, tagsRepoName := splitAndEncodeURLPath(tagsRepoPath)
+	pathToTags := fmt.Sprintf("https://api.github.com/repos/%v/%v/tags", tagsRepoOwner, tagsRepoName)
+	resp, err := http.Get(pathToTags)
+	if err != nil {
+		log.Fatal(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatal(err)
+	}
+	responseFromAPI := []byte(body)
+	var tags []tagInfo
+	err = json.Unmarshal(responseFromAPI, &tags)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return tags
+}
+
+func getChangeLogInfo(scanner *bufio.Scanner) changeLogInfo {
+	re, err := regexp.Compile(`## (?P<versionNumberAndDate>(?P<versionNumber>\d+\.\d+\.\d+).*)`)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Call [`regexp.MustCompile`](https://pkg.go.dev/regexp#MustCompile) instead, removing the need for the `if`:
```suggestion
	re, err := regexp.MustCompile(`## (?P<versionNumberAndDate>(?P<versionNumber>\d+\.\d+\.\d+).*)`)
```